src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +56,27 @@
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService = (MovieSessionService)
+                injector.getInstance(MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
                         fastAndFurious.getId(), LocalDate.now()));
+        AuthenticationService authenticationService = (AuthenticationService)
+                injector.getInstance(AuthenticationService.class);
+        try {
+            authenticationService.register(EMAIL, PASSWORD);
+            System.out.println("User registered successfully!");
+        } catch (RegistrationException e) {
+            System.out.println("Registration failed: " + e.getMessage());
+        }
+        try {
+            User user = authenticationService.login(EMAIL, PASSWORD);
+            System.out.println("User logged in: " + user);
```

Comment:
Ensure that the password is hashed before being stored or logged. Storing or logging passwords in plain text is a security risk.